### ISSUE TITLE
WTF 

### ISSUE THREAD

**Issue by chibenwa** [linagora/james-project#4617]
grep auth.upn.integration-open-paas.org * -R 

```
server/apps/cli-integration-tests/target/test-classes/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/cli-integration-tests/target/test-classes/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/cli-integration-tests/target/test-classes/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/cli-integration-tests/target/test-classes/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/cli-integration-tests/src/test/resources/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/cli-integration-tests/src/test/resources/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/cli-integration-tests/src/test/resources/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/cli-integration-tests/src/test/resources/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/memory-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/memory-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/memory-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/memory-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/cassandra-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/cassandra-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/cassandra-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/cassandra-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/distributed-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/distributed-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/distributed-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/distributed-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/distributed-pop3-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/distributed-pop3-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/distributed-pop3-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/distributed-pop3-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/jpa-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/jpa-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/jpa-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/jpa-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/jpa-smtp-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/jpa-smtp-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/apps/jpa-smtp-app/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/apps/jpa-smtp-app/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/mailet/integration-testing/target/classes/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/mailet/integration-testing/target/classes/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
server/mailet/integration-testing/src/main/resources/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
server/mailet/integration-testing/src/main/resources/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
third-party/spamassassin/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
third-party/spamassassin/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
third-party/spamassassin/sample-configuration/smtpserver.xml:                <oidcConfigurationURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/.well-known/openid-configuration</oidcConfigurationURL>
third-party/spamassassin/sample-configuration/smtpserver.xml:                <jwksURL>https://auth.upn.integration-open-paas.org/auth/realms/upn/protocol/openid-connect/certs</jwksURL>
```

DOD: 
 - same grep yield no result.
 - All OIDC sample configuration block is commented out.
 - See if 3.7.x is affected too

## Why ?

 -> Put a pet value instead of auth.upn.integration-open-paas.org
 -> Also this is a security backdoor, as we, Linagora, are able to authenticate on any james accounts if the config was not changes.

TLDR: it deserve a good spot on my security breach presentation

**Comment by vttranlina** [comment]
IMO, it is a public endpoint, when we use OIDC, any client can easy to get it

**Comment by chibenwa** [comment]
Let's discuss it in daily tomorrow.

Today we are able to craft access token allowing to login as ANY users for defaults james deployments (people not changing the smtp conf) and it is super bad.

Having an exemple (commented) is OK but we should avoid the backdoor in the default conf kind of trap!
